stream_alert/rule_processor/handler.py

```diff
@@ -133,6 +160,69 @@ def get_alerts(self):
         """
         return self._alerts
 
+    def _send_to_firehose(self):
+        """Put all classified records into its respective Firehose Delivery Stream
+
+        Args:
+            categorized_payloads (defaultdict): Categorized set of payloads by classified type
```

Comment:
I think this args list is outdated - it looks like you've made this an instance property instead

Reply:
Good catch